### ISSUE TITLE
Remove pre-meeting page on mobile

### ISSUE THREAD

**Issue by klinshy** [workadventure/workadventure#4460]
When entering a jitsi on mobile, you see this screen.

![Screenshot of App Brave Browser in Window CoCreationWorld Date 2024-12-13 at 13-37 001614](https://github.com/user-attachments/assets/d9a8c1ea-76d9-41d9-bcf6-e68e65b55099)

only after scrolling down you can see this small option to join from the browser, which you would wanna do eventually.

![Screenshot of App Brave Browser in Window New Issue · workadventureworkadventure 🔊 Date 2024-12-13 at 13-38 001618](https://github.com/user-attachments/assets/2c943d25-f0a5-4f26-bd3c-bce1f3a3b3f2)

**Comment by maestropsi** [comment]
Independent of what I do, I can't seem to disable that screen in WA. I can make this screen disappear on my own self-hosted Jitsi domain, but I can't get rid of it on WA. Can anyone help?
